core/src/main/java/org/jruby/RubyArray.java

```diff
@@ -1226,7 +1226,7 @@ protected SizeFn enumLengthFn() {
         final RubyArray self = this;
         return new SizeFn() {
             @Override
-            public IRubyObject size(IRubyObject[] args) {
+            public IRubyObject size(IRubyObject[] args, ThreadContext context, IRubyObject self1) {
```

Comment:
think it would be better to follow the 'general' order convention: `(context, self, args)`